.golangci.yml

```diff
@@ -89,10 +87,6 @@ linters-settings:
     # Select the Go version to target. The default is '1.13'.
     go: "1.21.13"
 
-  nakedret:
-    # make an issue if func has more lines of code than this setting and it has naked returns; default is 30
-    max-func-lines: 0
```

Comment:
`nakedret` is a linting check in both the `elastic-agent` and `beats` repos. To keep coding standards consistent across repos, perhaps it should be kept here as well and added to the enabled checks?

Reply:
To be clear, it *is* still enabled here. What this section does is set it to error for any function with a naked return. Removing it keeps it enabled, but at default configuration, which only applies to functions [longer than 30 lines](https://golangci-lint.run/usage/linters/#nakedret).

And yeah, imo we should do this everywhere. We recently did in elastic agent (https://github.com/elastic/elastic-agent/pull/5442).